L/libportaudio/build_tarballs.jl

```diff
@@ -31,15 +31,15 @@ if [ -d "asiosdk2.3.1" ]; then
     mv "asiosdk2.3.1 svnrev312937/ASIOSDK2.3.1" asiosdk2.3.1
 fi
 
-# Add the ringbuffer symbols needed by `pa_shim.c` to the windows export lists:
-IDX=80
-for SYM in PaUtil_GetRingBufferWriteAvailable \
-           PaUtil_GetRingBufferReadAvailable \
-           PaUtil_WriteRingBuffer \
-           PaUtil_ReadRingBuffer; do
-    echo "${SYM}    @${IDX}" >> ${WORKSPACE}/srcdir/portaudio/cmake_support/template_portaudio.def
-    IDX=$((IDX+1))
-done
+# apply the patch
+patch -dportaudio -p1 < portaudio_alsa_epipe_v3.diff
+
+# explicitly set the alsa env variables - for some reason
+# CMAKE_PREFIX_PATH wasn't working...
+if [ -f ${WORKSPACE}/destdir/lib/libasound.so]; then
```

Comment:
I'd use
```
${libdir}/libasound.${dlext}
```
instead of
```
${WORKSPACE}/destdir/lib/libasound.so
```

I think you also need a space before the `]`

Reply:
whoops - fixed...